### PR TITLE
Updates to the PR template

### PR DESCRIPTION
**Pull request by nghi-ly**
## Description & motivation
Update PR template to include link testing and the style guide

### CODE REVIEW

File: .github/pull_request_template.md
Comment (nghi-ly) [inline .github/pull_request_template.md]: want to confirm, is this the right URL to use for the style guide? thanks!

Comment (runleonarun) [inline .github/pull_request_template.md]: I think we need to use the absolute link to the `current` branch. We can test once Matt merges his PR
```suggestion
To learn more about the writing conventions used in the dbt Labs docs, see the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/contributing/content-style-guide.md). 
```

**Review by runleonarun — CHANGES_REQUESTED** [review]
I updated the link to the style guide in a comment. Otherwise this can be merged once we can test that link!

**Comment by nghi-ly** [comment]
@runleonarun : updated the link to the style guide to match the one used in PR https://github.com/dbt-labs/docs.getdbt.com/pull/2015 

good to go?

**Review by runleonarun — APPROVED** [review]
Looks great!

**Pull request merged**
